hasktorch/src/Torch/NN.hs

```diff
@@ -43,6 +44,14 @@ class Parameterized f where
   default replaceOwnParameters :: (Generic f, Parameterized' (Rep f)) => f -> ParamStream f
   replaceOwnParameters f = to <$> replaceOwnParameters' (from f)
 
+instance Parameterized Tensor where
+  flattenParameters _ = []
+  replaceOwnParameters = return
+
+instance Parameterized (Torch.Typed.Tensor.Tensor device dtype shape) where
+  flattenParameters _ = []
+  replaceOwnParameters = return
```

Comment:
I think this typed instance should be in Torch.Typed.Parameter.

Reply:
Thanks, I moved that one now.

Reply:
Thx!